sqlgen/db.go

```diff
@@ -55,17 +53,26 @@ func NewDB(conn *sql.DB, schema *Schema) *DB {
 				return nil, err
 			}
 			clause, args = selectQuery.ToSQL()
+			span.SetTag("query", clause)
+			span.SetTag("batchItemCount", len(items))
+			if marshaledItems, err := json.Marshal(filters); err == nil {
+				span.SetTag("batchItems", string(marshaledItems))
+			}
+			span.SetTag("queryVariables", args)
 
 			// Then, run the SQL query.
 			res, err := db.Conn.Query(clause, args...)
 			if err != nil {
+				ext.Error.Set(span, true)
+				span.SetTag("errorMessage", err.Error())
```

Comment:
I think we want to use span.LogFields(log.Error(err)) rather than defining our own errorMessage tag.